model_cards/neuraly/bert-base-italian-cased-sentiment/README.md

```diff
@@ -0,0 +1,101 @@
+---  
+language: it
+thumbnail: "https://neuraly.ai/static/assets/images/huggingface/thumbnail.png"  
```

Comment:
i think for social media the aspect ratio should be wider/more rectangle.

Maybe we should add guidelines somewhere (I know GitHub does it in repo settings)

Reply:
Oh ok cool! What about a 16:9 ratio? The URL remains the same though, its hosted on our website. Do you have any "template" reference?

Reply:
I changed the thumbnail with the attached image.
P.s.: The current version of the inference API is serving an older version of the model without labels. 
We fixed that, how long will it take before the API reloads the model config? Thank you 😊 

![thumbnail](https://user-images.githubusercontent.com/26767433/87799117-22329a00-c84d-11ea-9173-48b32ef9acfe.png)

Reply:
Looks awesome!

For the template, in any GitHub repo, you can look at the Settings > Social preview > Download template

Reply:
For the Inference API, should be updated now